apps/vr-tests-react-components/src/stories/DatePickerCompat.stories.tsx

```diff
@@ -87,4 +87,10 @@ storiesOf('DatePicker Compat', module)
     <Field label="Select a date">
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30281&commitId=0795ceb1edfc2a006830304b9e3e8cab5fc1d5b9&env=prod&clientType=FLUENTUI)


  <details><summary><b>DatePicker Compat</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|DatePicker Compat.when rendering inline, it should not render behind relatively positioned elements.default.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

Reply:
This looks good